content/blog/validating-smi-conformance-with-meshery.md

```diff
@@ -0,0 +1,70 @@
+
+From my time spent creating the v1.0 versions of other specifications like Redfish, Docker Benchmark, CloudEvents, and others, I can tell you that the promise of consistency lies at the heart of any specification. [SMI's specifications](https://github.com/servicemeshinterface/smi-spec) and their implementations are no different in this regard. Consistency enables portability and choice. Consistency enables great user experiences. Service Mesh Interface specifications present a consistent set of abstractions to describe functional service mesh configuration in a common way. SMI makes integrations with participating service meshes portable. In order to better serve these goals, the SMI community and Layer5 have developed an SMI Conformance initiative.
+
+## Conforming to SMI
+
+![SMI Conformance Program Overview](/img/blog/validating-smi-conformance-with-meshery/smi-comformance-initiative-overview.png)
+
+The scope of the [SMI Conformance](https://layer5.io/projects/service-mesh-interface-conformance) initiative includes all service mesh projects participating in the Service Mesh Interface specification. The goal of this initiative is to provide open tooling for validation of SMI specifications, instilling user confidence in the SMI implementation of their chosen service mesh. This validation initiative is not a full-blown certification program, yet, but has all of the makings to become one:
+
+- Versioned suite of conformance tests.
+- Self-reporting with guaranteed provenance and integrity of conformance test results.
+- Continually published conformance results.
+- Support for all service meshes that implement SMI.
+
+Delivering on these requirements required a tool specifically suited to SMI.
+
+![Meshery](/img/blog/validating-smi-conformance-with-meshery/meshery-logo-light-text-side.svg#center)
+
+## SMI Conformance Tool: Meshery
+
+Meshery, the service mesh management plane, is service mesh agnostic, which is an important characteristic as a compliance tool. No better tool for the job (in fact, no other tool capable of the job) than Meshery, which has been [a partner of SMI](https://meshery.io/blog/a-standard-interface-for-service-meshes) since its inception.
+
+![SMI Conformance Tool](/img/blog/validating-smi-conformance-with-meshery/meshery-service-mesh-compliance-tool.png).
+
+Meshery is ideal tooling in that it provides lifecycle management of a large number of service meshes and sample applications which need to be provisioned, tested, and deprovisioned in the process of validating conformance. And that Meshery is capable of generating load and verifying test assertions. Meshery will be extended to validate any service mesh that claims to implement and conform SMI specifications. And, more broadly, to leverage the same tooling to validate service mesh standards at-large, including SMI and [Service Mesh Performance](https://smp-spec.io) (SMP).
```

Comment:
> And that Meshery is capable of generating load and verifying test assertions.

And Meshery is capable of generating load and verifying test assertions.

(remove "that")

Reply:
> And, more broadly, to leverage the same tooling

I'd edit this to say "And, more broadly, Meshery will leverage the same tooling"

Reply:
Got it. ✔️